cmd/versitygw/main.go

```diff
@@ -369,6 +369,10 @@ func initFlags() []cli.Flag {
 }
 
 func runGateway(ctx context.Context, be backend.Backend) error {
+	if rootUserAccess == "" || rootUserSecret == "" {
+		return fmt.Errorf("root user access and secret key must be provided")
+	}
+
```

Comment:
Maybe to keep things simple and avoid additional checks we can set `Required` property to true in cli.Flag like below:
```
		&cli.StringFlag{
			Name:        "access",
			Usage:       "root user access key",
			EnvVars:     []string{"ROOT_ACCESS_KEY_ID", "ROOT_ACCESS_KEY"},
			Aliases:     []string{"a"},
			Required:    true,
			Destination: &rootUserAccess,
		},
		&cli.StringFlag{
			Name:        "secret",
			Usage:       "root user secret access key",
			EnvVars:     []string{"ROOT_SECRET_ACCESS_KEY", "ROOT_SECRET_KEY"},
			Aliases:     []string{"s"},
			Required:    true,
			Destination: &rootUserSecret,
		},
```

Reply:
The problem with this approach is that we would get an error when running the version option:
```
 % ./versitygw -v
2024/03/19 08:13:38 Required flags "access, secret" not set
```
I dont think this is what we want.

Reply:
Yes you are correct. There are other cases where we would get error like running the test command
```
./versitgw test -a access -s secret -e http://127.0.0.1:7070 full-flow
```